clusterloader2/testing/load/config.yaml

```diff
@@ -831,6 +831,7 @@ steps:
       useSimpleLatencyQuery: true
       summaryName: APIResponsivenessPrometheus_simple
       allowedSlowCalls: {{$ALLOWED_SLOW_API_CALLS}}
+      customThresholds: {{YamlQuote $CUSTOM_API_CALL_THRESHOLDS 4}}
```

Comment:
I don't understand this change - isn't customThreshold supposed to be a map?

Reply:
Explained offline but for posterity - it is `YamlQuote`, not `DefaultParam` ;).